pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[project]
+name = "gfw-data-api"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = "~= 3.10"
+dependencies = [
+    "aenum",
+    "aiofiles",
+    "aiohttp",
+    "alembic",
+    "arq",
+    "asgi_lifespan",
+    "async-lru",
+    "asyncpg",
+    "boto3",
+    "botocore",
+    "email-validator",
+    "fastapi",
+    "geoalchemy2<0.12",
+    "geojson",
+    "gino",
+    "gino_starlette",
+    "google-cloud-storage",
+    "httpcore",
+    "httpx",
+    "httpx-auth",
+    "newrelic",
+    "numpy<2",
+    "openapi_spec_validator",
+    "orjson",
+    "packaging",
+    "pendulum<3",
+    "pglast<2",
+    "psutil",
+    "psycopg2",
+    "pydantic<2",
+    "pyproj",
+    "python-multipart",
+    "retrying",
+    "shapely",
+    "sqlalchemy<1.4",
+    "sqlalchemy-utils",
+    "starlette",
+    "typer",
+    "uvicorn[standard]",
+]
+
+[dependency-groups]
+dev = [
```

Comment:
Seems like you dropped some packages under dev-packages in Pipfile, like asgi_lifespan, black, detect-secrets, and flake8.  Just checking, was that on purpose?

Reply:
I've gone back and forth on this, but they are unnecessary. They are required by pre-commit, but it installs its own copy of them. This is true of everything but asgi-lifespan, which I seem to have moved to the main dependency section. It should stay in the dev dependencies, as it's only used for tests. I will move back asgi-lifespan, but leave out the pre-commit packages.